servers/oidc.py

```diff
@@ -8,7 +8,7 @@
 from pathlib import Path
 
 HTTP_PROTOCOL = "http"
-
+OIDC_PORT = 8081
```

Comment:
Why change this?
Isn't i better to set our mock oidc under the `demo` profile, then there will be no conflicts with the LS-AAI mock

Reply:
I agree that some differentiation between the two profiles was needed, so I reverted the port change and split off the `demo` profile to its own compose file.